examples/text-generation/utils.py

```diff
@@ -573,6 +573,7 @@ def setup_generation_config(args, model, assistant_model, tokenizer):
     generation_config.flash_attention_causal_mask = args.flash_attention_causal_mask
     generation_config.flash_attention_fast_softmax = args.flash_attention_fast_softmax
     generation_config.trust_remote_code = args.trust_remote_code
+    setattr(generation_config, 'valid_sequence_lengths', None)
```

Comment:
@hsubramony  @ssarkar2  Isn't this where the default value is getting set? why do we get no attribute error?

Reply:
`valid_sequence_lengths` should be declared here: https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/generation/configuration_utils.py